driver/network_driver.go

```diff
@@ -79,6 +74,26 @@ func (d NetworkDriver) CreateNetwork(request *network.CreateNetworkRequest) erro
 		return err
 	}
 
+	// Calico driver does not support ipv6
+	ipv6, ok := request.Options["com.docker.network.enable_ipv6"].(bool)
```

Comment:
It doesn't look like this code is being hit from the test case - the error message is different?